[2025-04-09T07:28:02.528+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_spotify manual__2025-04-09T07:27:59.723185+00:00 [queued]>
[2025-04-09T07:28:02.543+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_spotify manual__2025-04-09T07:27:59.723185+00:00 [queued]>
[2025-04-09T07:28:02.544+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-09T07:28:02.594+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): extract_spotify> on 2025-04-09 07:27:59.723185+00:00
[2025-04-09T07:28:02.606+0000] {standard_task_runner.py:60} INFO - Started process 54 to run task
[2025-04-09T07:28:02.609+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'extract_spotify', 'manual__2025-04-09T07:27:59.723185+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmp018o2kct']
[2025-04-09T07:28:02.611+0000] {standard_task_runner.py:88} INFO - Job 88: Subtask extract_spotify
[2025-04-09T07:28:02.727+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.extract_spotify manual__2025-04-09T07:27:59.723185+00:00 [running]> on host d9666b3d8245
[2025-04-09T07:28:02.947+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='extract_spotify' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T07:27:59.723185+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T07:27:59.723185+00:00'
[2025-04-09T07:28:02.950+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T07:28:02.951+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/extract_spotify.py']
[2025-04-09T07:28:02.969+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T07:28:08.999+0000] {subprocess.py:93} INFO - error al extraer Spotify: [Errno 2] No such file or directory: '../data/raw/spotify/spotify.csv'
[2025-04-09T07:28:09.135+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T07:28:09.208+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=extract_spotify, execution_date=20250409T072759, start_date=20250409T072802, end_date=20250409T072809
[2025-04-09T07:28:09.358+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-09T07:28:09.396+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
